debian/control

```diff
@@ -4,7 +4,7 @@ Priority: optional
 Maintainer: Containerd team <help@containerd.io>
 Build-Depends: libbtrfs-dev | btrfs-tools ,
                debhelper,
-               dh-systemd,
+               debhelper (>= 9.20160709) | dh-systemd,
```

Comment:
Generally I would suggest ending this in a `~` so that backports will satisfy it (a backport of `9.20160709` would typically have a version number like `9.20160709~bpo90`), but in this case I wonder why we're keeping it so low?

Looking at https://packages.debian.org/search?keywords=debhelper and https://packages.ubuntu.com/search?keywords=debhelper, the only version of Debian that still has a version that's "too old" is the EOL Jessie release, and the only version of Ubuntu that's "too old" is Xenial (but backports has a new enough version, which is the route I'd recommend).

So here's what I'd suggest instead:

```suggestion
               debhelper (>= 10~),
```

(I think this also means you can ditch `--with systemd` in `debian/rules`)

Reply:
> Generally I would suggest ending this in a ~ so that backports will satisfy it (a backport of 9.20160709 would typically have a version number like 9.20160709~bpo90), but in this case I wonder why we're keeping it so low?

Ah, thanks; yes, I mostly copied the recommendation from https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=958585, but your recommendation definitely looks better

> but backports has a new enough version, which is the route I'd recommend

You mean adding the backports repo before running deb build, or would it have that enabled by default?

Reply:
I mean it's already enabled by default: :smile:

```console
$ docker run --rm ubuntu:xenial grep backports /etc/apt/sources.list
## Also, please note that software in backports WILL NOT receive any review
deb http://archive.ubuntu.com/ubuntu/ xenial-backports main restricted universe multiverse
# deb-src http://archive.ubuntu.com/ubuntu/ xenial-backports main restricted universe multiverse
```

Reply:
Ah, perfect 👍  updated